app.js

```diff
@@ -0,0 +1,125 @@
+
+//--file --validate --links --stats
+
+/*let fs = require('fs')
+function readFile() {
+    let index = process.argv.indexOf("--file")
+    if (index < 0) return console.log("No se encuentra el archivo")
+    let uri = process.argv[index + 1]
+    let fileContent = fs.readFileSync(uri, 'utf8')
+    console.log(process.argv)
+    console.log("index: ", index)
+    console.log("uri: ", uri)
+    console.log("contenido del archivo: ", fileContent)
+    readFile();
+ */
+
+    'use strict';
+var MarkDownIt = require('markdown-it'),
+    md = new MarkDownIt();
+let fs = require('fs');
+let path = require('path');
+let https = require('https');
+let http = require('http');
+const fetch = require('node-fetch');
+
+
+//find file
+function lookingForFile() {
+    let flag = process.argv.indexOf('--read')
+    let uri = process.argv[flag + 1]
+    console.log(uri);
+    let extUri = path.extname(uri)
+    if (extUri != '.md') {
+        console.log('Por favor ingresa un archivo con extensión .md');
+    } else {
+        readFile(uri)
+    }
+}
+
+//read file
+function scanFile(uri) {
+    let fileMd = fs.scanFileSync(uri, 'utf-8')
+    searchLinks(fileMd, uri)
+}
+
+
+//Get links
+function getLinks(fileMd, uri) {
+    let expRegURL = /^https?:\/\/[\w\-]+(\.[\w\-]+)+[/#?]?.*$/
+    let newFile = fileMd.replace(/[\(\)]/g, " ");
+    let expRegText = /(?<=\[).+?(?=\])/
+    let space = " "
+    let arrNewFile = newFile.split(space)
+    let links = arrNewFile.filter(text => text.includes('http'))
+    statsBasic(links)
+   
+
```

Comment:
It's not necessary to leave more than one blank space according to eslint. Also some semicolons are missing